README.md

```diff
@@ -580,7 +580,7 @@ defineInlineTest(transform, /* ... */)
 
 It's recommended that you set up your codemod project to all debugging via the VSCode IDE. When you open your project in VSCode, add the following configuration to your launch.json debugging configuration.
 
-```
+```json
```

Comment:
Can you please change it to `jsonc` (JSON with comments) instead? That should make it highlight the comments properly.

Reply:
Dear @Daniel15 ,

Thank you for your support. I updated. Please help me to review it.

Thanks,
Lam